frontend/src/pages/edit-driver-profile/components/edit-avatar-form/edit-avatar-form.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button, ImageSelector } from '~/libs/components/components.js';
+import { ImgPath } from '~/libs/enums/enums.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import {
+  useAppDispatch,
+  useAppForm,
+  useCallback,
+  useMemo,
+  useRef,
+} from '~/libs/hooks/hooks.js';
+import { useAuthUser } from '~/slices/auth/auth.js';
+import { uploadAvatar } from '~/slices/files/actions.js';
+
+import { type EditAvatarFormData } from './libs/types/types.js';
+import styles from './styles.module.scss';
+
+const AVATAR_SIZE = 200;
```

Comment:
there are two different constant AVATAR_SIZE in 2 files, one =180, other = 200. Let's add more proper name, and move them to some constants file